charts/timescaledb-multinode/values.yaml

```diff
@@ -14,11 +12,6 @@ image:
   tag: pg12-ts2.0.0-p0
   pullPolicy: IfNotPresent
 
-#enable external access using LoadBalancer
-accessNode:
-  service:
-    type: LoadBalancer
-
 # Credentials used by PostgreSQL
 credentials:
   fromValues: true
```

Comment:
should I also move these to each node ?

Reply:
aren't they belong to access node only ?
if so I guess they better stay here

Reply:
both access node and data node use the credentials

Reply:
ohh
well good question
not sure then :-)